src/unity/python/turicreate/data_structures/sframe.py

```diff
@@ -4973,7 +4976,31 @@ def unpack(self, column_name, column_name_prefix=None, column_types=None,
         | 3  |    3.0    |    4.0    |    5.0    |
         +----+-----------+-----------+-----------+
         [3 rows x 4 columns]
+        
+        >>> sf = turicreate.SFrame([{'a':1,'b':2,'c':3},{'a':4,'b':5,'c':6}])
+        >>> sf.unpack()
+        +---+---+---+
+        | a | b | c |
+        +---+---+---+
+        | 1 | 2 | 3 |
+        | 4 | 5 | 6 |
+        +---+---+---+
+        [2 rows x 3 columns]
+
+
+
         """
+        if column_name==None:
```

Comment:
Since there is only one `None` value, generally `is` is preferred when checking against `None`. 
So this is probably better: `if column_name is None:`